source/class/cv/IconHandler.js

```diff
@@ -235,14 +235,22 @@ qx.Class.define('cv.IconHandler', {
      * @returns {string}
      */
     getIconSource: function (name, classes) {
-      const i = this.get.apply(this, arguments);
+      const i = this.get(name);
       if (i) {
         if (!classes) {
           classes = 'icon';
         }
         if (typeof i === 'function') {
-          return i(undefined, undefined, classes, true);
-        } 
+          const res = i(undefined, undefined, classes, true);
+          if (res.startsWith('<canvas')) {
+            // no support for canvas as icon preview
+            return '';
+          }
+          return res;
+        }
+        if (/\.svg#.*?$/.test(i.uri)) {
```

Comment:
Is it importan here to have a SVG with fragment, i.e. `/\.svg#.*?$/`?
Or should this test be true for all SVG, i.e. `/\.svg(#.*)?$/`?

Reply:
"Normal"-SVGs can be loaded with <img src="..."/> so no special treatment is necessary for them, they work like any other image (png, jpg,...)